label_studio/jwt_auth/auth.py

```diff
@@ -50,4 +50,9 @@ def get_security_definition(self, auto_schema):
             '<br><pre><code class="language-bash">'
             'curl https://label-studio-host/api/projects -H "Authorization: Token [your-token]"'
             '</code></pre>',
+            'x-fern-header': {
+                'name': 'api_key',
+                'env': 'LABEL_STUDIO_API_KEY',
+                'prefix': 'Token ',
```

Comment:
One thing I never figured out is... 
api_key param supports both legacy token, and jwt. The docs specify -H "Authorization: Token [your-token]" but it needs to be `Bearer` when using jwt. Confusing for users. Not sure if there is a clean way to show both 

https://humansignal.slack.com/archives/C072ZLBQ6SJ/p1749753981577629?thread_ts=1749750510.009489&cid=C072ZLBQ6SJ

Reply:
I think we could figure this out, but I wanted to keep consistent with what we had before for this initial version